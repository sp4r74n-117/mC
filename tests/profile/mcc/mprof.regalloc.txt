function main
 called from ??
 took avg: 2124764 stddev: 0 cycles
function sum
 called from main
 took avg: 21187 stddev: 2607 cycles
