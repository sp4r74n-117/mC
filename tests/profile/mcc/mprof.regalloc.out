0x80489fd 0x8048979 20144
0x80489fd 0x8048979 21136
0x80489fd 0x8048979 21352
0x80489fd 0x8048979 20352
0x80489fd 0x8048979 21636
0x80489fd 0x8048979 21396
0x80489fd 0x8048979 22064
0x80489fd 0x8048979 20076
0x80489fd 0x8048979 20192
0x80489fd 0x8048979 20072
0x80489fd 0x8048979 21628
0x80489fd 0x8048979 22312
0x80489fd 0x8048979 20124
0x80489fd 0x8048979 20080
0x80489fd 0x8048979 21672
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 21568
0x80489fd 0x8048979 21428
0x80489fd 0x8048979 21564
0x80489fd 0x8048979 21428
0x80489fd 0x8048979 21564
0x80489fd 0x8048979 21428
0x80489fd 0x8048979 22048
0x80489fd 0x8048979 22372
0x80489fd 0x8048979 20304
0x80489fd 0x8048979 22040
0x80489fd 0x8048979 20208
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 21896
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 22040
0x80489fd 0x8048979 22148
0x80489fd 0x8048979 20632
0x80489fd 0x8048979 20608
0x80489fd 0x8048979 20632
0x80489fd 0x8048979 20608
0x80489fd 0x8048979 20632
0x80489fd 0x8048979 20100
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20080
0x80489fd 0x8048979 22152
0x80489fd 0x8048979 21428
0x80489fd 0x8048979 21564
0x80489fd 0x8048979 21428
0x80489fd 0x8048979 20844
0x80489fd 0x8048979 22080
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 21560
0x80489fd 0x8048979 20632
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20096
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20096
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20096
0x80489fd 0x8048979 22232
0x80489fd 0x8048979 22148
0x80489fd 0x8048979 45716
0x80489fd 0x8048979 22108
0x80489fd 0x8048979 21892
0x80489fd 0x8048979 20860
0x80489fd 0x8048979 22188
0x80489fd 0x8048979 20320
0x80489fd 0x8048979 20080
0x80489fd 0x8048979 20124
0x80489fd 0x8048979 21512
0x80489fd 0x8048979 20928
0x80489fd 0x8048979 20772
0x80489fd 0x8048979 20600
0x80489fd 0x8048979 22300
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 20332
0x80489fd 0x8048979 22040
0x80489fd 0x8048979 20980
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 22080
0x80489fd 0x8048979 22364
0x80489fd 0x8048979 20612
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 20108
0x80489fd 0x8048979 20080
0x80489fd 0x8048979 20092
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 20088
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20084
0x80489fd 0x8048979 22120
0x80489fd 0x8048979 20088
0x804894e 0xf758d72e 2124764
